Resources/Prototypes/_Nuclear14/Entities/Structures/Furniture/rugs_carpets.yml

```
@@ -0,0 +1,169 @@
+  abstract: true
+  components:
+  - type: Sprite
+    drawdepth: FloorTiles
+  - type: Physics
+    canCollide: false
+  - type: Fixtures
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Destructible
+    thresholds:
+      - trigger:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)